### ISSUE TITLE
Make greater than or equal version work instead of static good version

### ISSUE THREAD

**Issue by jpadams** [puppetlabs/razor-ppc64le-mk#5]
```
power8-alpine-1:~/razor-ppc64le-mk# uname -r
4.14.49-0-vanilla
power8-alpine-1:~/razor-ppc64le-mk# ./setup-razor-env.sh

Verifying kernel version...

Please update kernel to at least 4.14.48-0-vanilla
```

**Comment by mtarsel** [comment]
This issue should be resolved once the final v3.8 release of alpine is published. We will use that version of alpine going forward and either:
1. not check the kernel version at all
2. convert the version into a long int without periods and verify its the right version

**Comment by mtarsel** [comment]
This script now uses the official Alpine 3.8 netboot images for ppc64le. See 707d8704c09b93a23a5e3f4c4359e791321cd37d and previous commits from there
